src/Dialect/Krnl/DialectBuilder.cpp

```diff
@@ -126,8 +126,8 @@ ValueRange KrnlBuilder::getInductionVarValue(ValueRange loops) const {
       .getResults();
 }
 
-void KrnlBuilder::parallel(Value loop) const {
-  b().template create<KrnlParallelOp>(loc(), loop);
+void KrnlBuilder::parallel(ValueRange loops) const {
```

Comment:
If it does not generate conflicts, you may want to consider a constructor that also take a single Value.

Reply:
ValueRange will be automatically constructed when a single Value is passed. You may have noticed that the current code to call this builder function remains the same.